hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimeGeneratorBase.java

```diff
@@ -79,24 +71,26 @@ public TimeGeneratorBase(HoodieTimeGeneratorConfig config, SerializableConfigura
     this.lockConfiguration = config.getLockConfiguration();
     this.hadoopConf = hadoopConf;
 
-    maxRetries = lockConfiguration.getConfig().getInteger(LOCK_ACQUIRE_CLIENT_NUM_RETRIES_PROP_KEY,
+    // The maximum times to retry in case there are failures.
+    int maxRetries = lockConfiguration.getConfig().getInteger(LOCK_ACQUIRE_CLIENT_NUM_RETRIES_PROP_KEY,
         Integer.parseInt(DEFAULT_LOCK_ACQUIRE_NUM_RETRIES));
     lockAcquireWaitTimeInMs = lockConfiguration.getConfig().getInteger(LOCK_ACQUIRE_WAIT_TIMEOUT_MS_PROP_KEY,
         DEFAULT_LOCK_ACQUIRE_WAIT_TIMEOUT_MS);
-    maxWaitTimeInMs = lockConfiguration.getConfig().getLong(LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS_PROP_KEY,
+    // The maximum time to wait for each time generation to resolve the clock skew issue on distributed hosts.
+    long maxWaitTimeInMs = lockConfiguration.getConfig().getLong(LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS_PROP_KEY,
         Long.parseLong(DEFAULT_LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS));
     lockRetryHelper = new RetryHelper<>(maxWaitTimeInMs, maxRetries, maxWaitTimeInMs,
         Arrays.asList(HoodieLockException.class, InterruptedException.class), "acquire timeGenerator lock");
   }
 
-  protected LockProvider getLockProvider() {
+  protected LockProvider<?> getLockProvider() {
```

Comment:
why this change here? maybe do in a separate commit?

Reply:
Yeah, the change is not relevent.